lib/src/components/NxTransferListHalf/NxTransferListHalf.scss

```diff
@@ -104,22 +104,29 @@ $-visible-item-count: 10;
 
   border-radius: var(--nx-border-radius);
   border: var(--nx-border-default);
-  cursor: pointer;
   flex: auto;
   padding: var(--nx-spacing-2x);
+  padding-left: var(--nx-spacing-4x);
```

Comment:
Why is this 4x? Shouldn't it be 2x whether the icon is there or not?

Reply:
> Why is this 4x? Shouldn't it be 2x whether the icon is there or not?

I see that this does match the mockup.  However I recall a discussion with the designers where I believe they confirmed the spacing would be the same with and without the icon.  So unless you've doubled checked this with them already, I'm wondering if it's a mistake in the mockup.

Reply:
I confirmed with designs @JasonDSwearingen, the padding would be 4x when rendered without the icon.